src/components/Legend/Element.tsx

```diff
@@ -18,7 +18,10 @@ const Element = ({element, color}: ElementProps) => {
         style={{
           backgroundColor: techClicked === element.index ? color : '#fff',
         }}
-        onClick={() => setTechClicked(element.index)}>
+        onClick={() => {
+          element.link ? window.open(element.link) : null;
```

Comment:
nie jestem pewien czy otwieranie nowego okna na kliku to dobre zachowanie, może lepiej było by zobaczyć listę linków np

Reply:
Zrobiłam tak jak jest na Zalando Radar (https://opensource.zalando.com/tech-radar/) i słyszałam głosy że dobrze żeby to było na clicku - do przegadania ponownie